CHANGELOG.md

```diff
@@ -314,6 +315,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### State Machine Breaking
 
+* (x/staking) [#15701](https://github.com/cosmos/cosmos-sdk/pull/15701) The `HistoricalInfoKey` has been updated to use a binary format.
```

Comment:
This is in the wrong section. Move it up to the unreleased section above.

Reply:
Thanks, fixed